aircraft-system.js

```
@@ -326,51 +358,186 @@ export class AircraftSystem {
-        // Add some dramatic rotation
-        this.aircraft.rotation.x = Math.PI * 0.3;
-        this.aircraft.rotation.z = Math.PI * 0.2;
+        // NEW: Spatially aware crash mechanics based on collision part and impact velocity
+        const crashQuaternion = new THREE.Quaternion();
+        const impactIntensity = Math.min(crashVelocity / 50, 1.0); // Normalize impact intensity
+        
+        switch(collisionPart) {
+            case 'leftWing':
+                // Left wing strike causes aircraft to cartwheel left and nose down
+                this.aircraft.position.y = terrainHeight + 3; // Lower to ground
+                const leftWingPitch = Math.PI * (0.15 + impactIntensity * 0.25); // 15-40 degrees nose down
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)